### PR TITLE
Show core experience to old TLS folks

### PR DESCRIPTION
**Pull request by lc512k**
Anyone with TLS under 1.1 will get the "Your browser is not supported" page. We can add a custom message so it varies slightly if we need to. #421 

Moved the core experience / tls checkups to its own controller

cc @commuterjoy @ifyio

### CODE REVIEW

File: client/routes/core-experience/controller.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		function showBrowserNotSupported () {
+			document.querySelector('#core').style.display = 'block';
+			document.querySelector('#enhanced').style.display = 'none';
+		}
+
+		if (!window.cutsTheMustard) {
+			showBrowserNotSupported();
+		}
+
+		fetch('https://www.howsmyssl.com/a/check')
```
Comment (ifyio) [inline client/routes/core-experience/controller.js]: Can this API be called if TLS is disabled on the client browser? Or maybe we don't care about that?

Comment (ifyio) [inline client/routes/core-experience/controller.js]: Maybe we should have some error handling of some sort

Comment (ifyio) [inline client/routes/core-experience/controller.js]: Should in case the API is down

Comment (lc512k) [inline client/routes/core-experience/controller.js]: Catching the errors now

**Pull request merged**